deployment/components/master-config/nfd-master.conf.example

```diff
@@ -5,3 +5,7 @@
 # enableTaints: false
 # labelWhiteList: "foo"
 # resyncPeriod: "2h"
+# leaderElection:
+#   leaseDuration: 15s
+#   renewDeadline: 10s
+#   retryPeriod: 2s
```

Comment:
Thanks for adding these. My only small concern is overwhelming the user with detailed configuration options. Is there any advice we could give or point to (in the documentation) for users to be able to pick meaningful values. Like "renewDeadline should be shorter than leaseDuration", "retryPeriod should be shorter than renewDeadline"... Also, would it make sense to have proportional/dynamic defaults, i.e. renewDeadline would be 2/3*leaseDuration and retryPeriod renewDuration/5 if nothing is specified in those fields?

Any thoughts, WDYT?

Reply:
Right now those values are being defaulted to what is recommended in documentation for leader election.
I can add some comments here what are the rules of specifying them in example conf.

Reply:
> I can add some comments here what are the rules of specifying them in example conf

That would be great 🙏 If you do, please add the same comments to the config-reference in the docs, too